content/knowledge-codemagic/build-inputs.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Build inputs
+description: Specify optional inputs that are passed to the workflow when starting a build
+weight: 3
+---
+
+## Overview
+
+Build inputs are customizable parameters you can define within your workflow to make it more adaptable to different scenarios. With build inputs, you can create a single workflow and run it with different configurations by providing the values for inputs when starting a build for the workflow. For example, you can use build inputs to determine whether to build the workflow for test or release purposes or which app flavor to build. This eliminates the need to create multiple similar workflows with specific hardcoded values, making the workflow more reusable and dynamic.
+
+Inputs are workflow-specific and are defined in `codemagic.yaml` under the `inputs` mapping (see the [example](#minimal-example) below). The started workflow receives specified input values in the `inputs` context, i.e. `${{ inputs.inputId }}` is replaced with the value passed to input with identifier `inputId`.
+
+### Minimal example
+
+This example configures one input with the ID `name`. Unless given another value when starting a build, `name` defaults to `Codemagic`.
+
+{{< highlight yaml "style=paraiso-dark">}}
+workflows:
+  greetings:
+    inputs:
+      name:
+        description: Who is greeted?
+        default: Codemagic
+    scripts:
+      - echo "Hello, ${{ inputs.name }}"
+{{< /highlight >}}
+
+## Starting builds with Inputs
+
+All inputs must be specified to successfully start a build, either by providing a `default` value in the YAML configuration, or giving a one-off value when starting the build. 
```

Comment:
this text duplicated and sounds better in the next section.

I suggest to remove this header and move 2 next sections to 1 level up:
1. Start builds with Inputs manually
2. Start builds with Inputs automatically